stable/insights-agent/templates/release-watcher/rbac.yaml

```diff
@@ -1,14 +1,43 @@
 {{- if .Values.releasewatcher.enabled -}} 
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ include "insights-agent.fullname" . }}-release-watcher
+  name: {{ include "insights-agent.fullname" . }}-release-watcher-view
   labels:
     app: insights-agent
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: edit
+  name: view
```

Comment:
Where is this `ClusterRole` coming from?

Reply:
View? It's a built in ClusterRole, I mimicked the changes to the release-watcher helm chart to lower the amount of permissions we're taking.

Reply:
Aha! I knew I was missing something.